custodia/secrets.py

```diff
@@ -259,6 +259,8 @@ def _int_get_key(self, trail, request, response):
             output = self.root.store.get(key)
             if output is None:
                 raise HTTPError(404)
+            elif len(output) == 0:
+                raise HTTPError(406)
```

Comment:
I do not think we should support empty values, but if we want to do that then we need to change the DB schema to mark what's a container, because in the sqlite db the only difference right now is that a container key has an empty value.